myhpom/templates/myhpom/accounts/next_steps.html

```diff
@@ -35,9 +36,11 @@ <h5>
                                             Download a Template Advance Directive
                                         </a>
                                     </h5>
+                                    {% scribble 'next-steps-download' 'shared' %}
                                     <p>
```

Comment:
We don't want to encourage anyone to style with this change, just plug in content. I think you should put these <p> tags outside the scribble area to prevent anyone thinking they're supposed to be able to do that.

Reply:
Actually looking down at the rest that have some stray <p> or or TMs in them, they seem appropriate. I guess its just the 'FAQ' content that has a <p> in it where I don't think we'd want the user to put any more than tho one paragraph. Just tweak that one?

Reply:
Yes, I agree with changing the dashboard FAQs and leaving the rest. Thanks!